docs/sample-solutions/windows-server/healthcare-multitier-app.md

```diff
@@ -0,0 +1,111 @@
+* [Azure Windows virtual machines][vm-docs] provide core compute resources for Windows Server VM instances. A VM instance is deployed in the backend tier that acts as a fileshare witness for the SQL Server cluster.
+
+### Availability
+
+For other scalability topics, see the [availability checklist][availability] available in the architecure center.
+
+### Scalability
+
+For other scalability topics, see the [scalability checklist][scalability] available in the architecure center.
+
+### Security
+
+For a deeper discussion on [security][], see the relevant article in the architecture center.
+
+### Resiliency
+
+For a deeper discussion on [resiliency][], see the relevant article in the architecture center.
+
+## Deploy the solution
+
+**Prerequisites.**
+
+* You must have an existing Azure account. If you don't have an Azure subscription, create a [free account](https://azure.microsoft.com/free/?WT.mc_id=A261C142F) before you begin.
+
+To deploy this solution with an Azure Resource Manager template, perform the following steps.
+
+1. Select the **Deploy to Azure** button:<br><a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fiainfoulds%2Farchitecture-center%2Fwindowsserver%2Fdocs%2Fsample-solutions%2Fwindows-server%2Ftemplates%2Fhealthcare-multitier-app%2Fazuredeploy.json" target="_blank"><img src="http://azuredeploy.net/deploybutton.png"/></a>
+2. Wait for the template deployment to open in the Azure portal, then complete the following steps:
+   * Choose to **Create new** resource group, then provide a name such as *myWindowsSolution* in the text box.
+   * Select a region from the **Location** drop-down box.
+   * Provide a username and secure password for the Jenkins instance and Grafana console.
+   * Review the terms and conditions, then check **I agree to the terms and conditions stated above**.
+   * Select the **Purchase** button.
+
+It can take 15-20 minutes for the deployment to complete.
+
+## Pricing
+
+To explore the cost of running this solution, all of the Azure service components are pre-configured in the following cost calculator links.  To see how the pricing would change for your particular use case, change the appropriate variables to match your expected traffic.
+
+We have provided three sample cost profiles based on the number of scale set VM instances that run your applications.
+
+* [Small][small-pricing]: this correlates to two frontend and two backend VM instances.
+* [Medium][medium-pricing]: this correlates to 20 frontend and 5 backend VM instances.
+* [Large][large-pricing]: this correlates to 100 frontend and 10 backend VM instances.
+
+## Related Resources
+
+This solution used a backend virtual machine scale set that runs a Microsoft SQL Server cluster. Azure Cosmos DB could also be used as a scalable and secure database tier for the application data. For more information, see [Azure Cosmos DB overview][azurecosmosdb-docs].
```

Comment:
I guess Cosmos DB for a healthcare scenario should use a [VNet service endpoint](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-service-endpoints-overview)?

Reply:
Added sentence to specify the use of VNet service endpoint and link to doc.